alure.pxd

```diff
@@ -24,6 +24,16 @@ from libcpp.utility cimport pair
 from libcpp.vector cimport vector
 
 
+# std::chrono wrapper
+cdef extern from '<chrono>' namespace 'std' nogil:
+    cdef int nanoseconds
+    cdef int microseconds
+    cdef int milliseconds
+    cdef int seconds
+    cdef int minute
+    cdef int hour
```

Comment:
Notice `std::chrono::duration`
```c++
template <class Rep, class Period = ratio<1>>
class duration {
public:
  using rep = Rep;
  // nullary construct/destroy
  constexpr duration() = default;
  ~duration() = default;
  // observer
  constexpr rep count() const;
};
```
while convenience typedefs are defined as
```c++
using nanoseconds = duration</*signed integer type of at least 64 bits*/ , nano>;
using microseconds = duration</*signed integer type of at least 55 bits*/ , micro>;
using milliseconds = duration</*signed integer type of at least 45 bits*/ , milli>;
```
where `ratio`s are defined [here](https://en.cppreference.com/w/cpp/numeric/ratio/ratio).

Notice `Seconds` is defined in `alure2-aliases.h` as
```c++
using Seconds = std::chrono::duration<double>;
```

In short, the conversion would be

C++ type | Cython code | Python type
--- | --- | ---
`nanoseconds n` | `n.count()` (type `uint64_t`) | `int`
`Seconds x` | `x.count()` (type `double`) | `float`

Reply:
OOF `.+seconds.rep` should be signed, so use `int64_t` instead of `uint64_t`.